app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        Person.people[name] = self
```

Comment:
Storing the `Person` instance in the `people` dictionary using the name as the key is correct. Ensure that names are unique to avoid overwriting existing entries.